admin_guide/pruning_resources.adoc

```diff
@@ -33,10 +33,58 @@ $ oc adm prune <object_type> <options>
 
 This specifies:
 
-- The `<object_type>` to perform the action on, such as `builds`,
+- The `<object_type>` to perform the action on, such as `groups`, `builds`,
 `deployments`, or `images`.
 - The `<options>` supported to prune that object type.
 
+[[pruning-groups]]
+
+== Pruning groups
+
+In order to prune groups records from an external provider, administrators may
+run the following command:
+
+---
+$ oc adm prune groups --sync-config=path/to/sync/config [<options>]
+---
+
+.Prune Groups CLI Configuration Options
+[cols="4,8",options="header"]
+|===
+
+|Options |Description
+
+.^|`--confirm`
+|Indicate that pruning should occur, instead of performing a dry-run.
+
+.^|`--blacklist`
+|Path to the group blacklist file.
+See xref:../install_config/syncing_groups_with_ldap.adoc#overview[Syncing Groups With LDAP]
+for the structure of this file.
+
+.^|`--whitelist`
+|Path to the group whitelist file.
+See xref:../install_config/syncing_groups_with_ldap.adoc#overview[Syncing Groups With LDAP]
+for the structure of this file.
+
+.^|`--sync-config`
+|Path to the synchronization configuration file.
+See xref:../install_config/syncing_groups_with_ldap.adoc#overview[Syncing Groups With LDAP]
```

Comment:
I'd link to that section: ../install_config/syncing_groups_with_ldap.html#configuring-ldap-sync[Configuring LDAP Sync]